lib/components/app/call-taker-panel.js

```diff
@@ -285,8 +289,13 @@ class CallTakerPanel extends Component {
             <button
               className='clear-button-formatting'
               style={{marginBottom: '5px', marginLeft: '10px'}}
+              disabled={addIntermediateDisabled}
               onClick={this._addPlace}>
-              <i className='fa fa-plus-circle' /> Add place
+              <i className='fa fa-plus-circle' />{' '}
+              {addIntermediateDisabled
+                ? 'Define origin/destination to add intermediate places'
```

Comment:
This message will display also when the max number of intermediate places has been reached.